### PR TITLE
Document processing of local data using python transform image

### PR DESCRIPTION
**Pull request by daw3rd**
## Why are these changes needed?
Makes it easy to process local data.

## Related issue number (if any).
#317

### CODE REVIEW

**Review by roytman — COMMENTED** [review]
Why do you add this mounted directories to pure python only, it is useful for Ray local as well.

**Review by roytman — APPROVED** [review]
LGTM

File: transforms/code/code2parquet/python/Dockerfile [hunk 1]
```diff
@@ -10,6 +10,9 @@ RUN useradd -ms /bin/bash dpk
 USER dpk
 WORKDIR /home/dpk
 
+# Create directories to mount volumnes for processing data outside of the image.
+RUN mkdir input && mkdir output
```
Comment (blublinsky) [inline transforms/code/code2parquet/python/Dockerfile:14]: I am not convinced we need this. According to https://www.warp.dev/terminus/docker-run-volume you can mount any local directory into a Docker container and then use it for execution. There is no need to do mkdir. Can you please verify this

File: transforms/code/code2parquet/python/Dockerfile [hunk 2]
```diff
@@ -10,6 +10,9 @@ RUN useradd -ms /bin/bash dpk
...
+
```
Comment (blublinsky) [inline transforms/code/code2parquet/python/Dockerfile]: I am not convinced we need this. According to https://www.warp.dev/terminus/docker-run-volume you can mount any local directory into a Docker container and then use it for execution. There is no need to do mkdir. Can you please verify this. Even more info is here https://docs.docker.com/storage/volumes/

**Review by blublinsky — CHANGES_REQUESTED** [review]

File: transforms/code/code2parquet/python/Dockerfile (hunk 2 above)
Comment (daw3rd) [reply transforms/code/code2parquet/python/Dockerfile]: Ok yes, no need to create the mount points ahead of time.  Reverted the Dockerfiles and updated the transform readmes to point to the quickstart for running images.  Thanks.

**Review by blublinsky — APPROVED** [review]

**Pull request merged**